install/windows.md

```diff
@@ -629,17 +487,161 @@ TeXLive zu installieren.
 
 __Jetzt noch die Installation [testen](#test)!__
```

Comment:
Rausnehmen, sonst hat keiner VSCode und Sumatra.

Reply:
In the mamba section we also have:
> __Jetzt noch die Python-Installation [updaten](#update) und anschließend [testen](#test)!__
Im going to remove this to